data/strings/strings.txt

```diff
@@ -27357,6 +27976,7 @@
     ja = ガソリンスタンド
     ko = 주유소
     lt = Degalinės
+    lv = DUS
```

Comment:
"Degvielas uzpildes stacijas"

Reply:
Yes, if length is not an issue, I agree to change.